src/elements/Chip/DeletableChip.tsx

```diff
@@ -0,0 +1,95 @@
+import { FC, forwardRef, Ref, ReactElement } from 'react';
+import classNames from 'classnames';
+import { Button } from '../Button';
+import { ChipProps } from './Chip';
+import { CloseIcon } from '../../form/Select/icons';
+import css from './Chip.module.css';
+
+export interface DeletableChipProps extends ChipProps {
+  /**
+   * Content to display before the chip label.
+   */
+  start?: ReactElement | string;
+
+  /**
+   * Customize delete icon.
+   */
+  deleteIcon?: ReactElement;
+
+  /**
+   * Callback if the chip is deletable.
+   */
+  onDelete?: () => void;
+}
+
+export interface ChipRef {
+  ref?: Ref<HTMLDivElement>;
+}
+
+export const DeletableChip: FC<DeletableChipProps & ChipRef> = forwardRef(
+  (
+    {
+      children,
+      color,
+      variant,
+      size,
+      start,
+      selected,
+      disabled,
+      className,
+      disableMargins,
+      deleteIcon,
+      onClick,
+      onDelete,
+      ...rest
+    },
+    ref: Ref<HTMLDivElement>
```

Comment:
@steppy452 - Is there a reason we can't re-use the chip and just pass these as props to the `end`?

Reply:
ok yea reusing the chip makes a lot of sense. but not sure `end` will be what we want since there is an onDelete callback for the delete button. unless you're saying we should put `start` and `end` on the regular Chip component. but then it feels a little silly to have a wrapper component for deletable since the structure will be the same

Reply:
Yes, I was meaning export this as a delete chip but internally it uses the 'end' and omits that from the available types. This is really just a shortcut component.